src/pages/Dashboard/index.tsx

```diff
@@ -18,7 +17,7 @@ const Dashboard = () => {
 
       <TopContainer>
         <StatsWrapper>
-          <PortfolioWidget />
+          <Portifolio />
```

Comment:
Loading icon is not properly visible. dots around PNG icon is not coming properly

https://user-images.githubusercontent.com/6604146/171563767-0eb8f12e-10f0-4201-993a-29822e5762a1.mov